tensorflow/contrib/cmake/python_module_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+from __init__ import rm_init_py
+rm_init_py(__file__)
+
+import test_commons as commons
```

Comment:
(I do not think we need this as a separate file, but)
why not import this as
```
from tensorflow.contrib.cmake import test_commons
```

Reply:
> I do not think we need this as a separate file

The tests share a good chunk of their logic, which I de-duplicated to make it usable by potential additional tests.

As for the import:
This is an artifact of experiments I did, due to issues with `__file__`.
But now, I can change that.

Reply:
What I mean is the tests are small enough that we can merge them to exist in a single file.

Reply:
Okay, will do.